.github/actions/test-templates/action.yaml

```diff
@@ -3,39 +3,163 @@ name: Test Templates
 runs:
   using: "composite"
   steps:
+
+  # Protobuf Client template
+
   - name: 🧪 Test IceRpc + Protobuf Client Template
-    run: mkdir -p templates-test/ProtobufClient && cd templates-test/ProtobufClient && dotnet new icerpc-protobuf-client && dotnet build /p:TreatWarningsAsErrors=true
+    run: |
+      dotnet new icerpc-protobuf-client -o templates-test/tcp/ProtobufClient
```

Comment:
using `dotnet new -o` to create the directories solves the issue about the action not working on Windows